File: Java/MultiplicationTableBug.java
package Java;

public class MultiplicationTableBug {
    public static void printMultiplicationTable(int n) {
        for (int i = 0; i < 5; i++) {
            System.out.println(n + " x " + i + " = " + n);
        }
    }

    public static void main(String[] args) {
        int number = 5;
        printMultiplicationTable(number);
    }
}
// Bug 1: Incorrect loop range
// Bug 2: Incorrect multiplication
// Bug 3: Hardcoded single number

File: C/MultiplicationTableBug.c
#include <stdio.h>

#include <stdio.h>

void printMultiplicationTable(int n)
{
    for (int i = 0; i < 5; i++)
    {
        printf("%d x %d = %d\n", n, i, n);
    }
}

int main()
{
    int number = 5;
    printMultiplicationTable(number);
    return 0;
}
// Bug 1: Incorrect loop range
// Bug 2: Incorrect multiplication
// Bug 3: Hardcoded single number